src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -64,6 +65,27 @@ export const TradeConfirm = ({ history }: RouterProps) => {
 
   const status = useAppSelector(state => selectLastTxStatusByAssetId(state, caip))
 
+  // Parametrized errors cannot simply be matched with === since their param(s) might vary
+  const PARAMETRIZED_ERRORS_TO_TRADE_ERRORS = {
```

Comment:
Params seems not to be relevant to add to toast messages e.g:
```
TradeConfirm:onSubmit - Error: ZrxExecuteQuote - signAndBroadcastTransaction error: Error: Error signing & broadcasting tx
```

The parametrized part is basically just the same as the base message. However, this error is the only signing one that I have tested with a real scenario (rejecting transaction) and the others ones were thrown by monkey-patched `swapper` build. By clearly separating parametrized errors from constant errors, we will be able to easily handle parameters in the future if needed.